### ISSUE TITLE
make the design responsive

### ISSUE THREAD

**Issue by daonb** [hasadna/open-shot#107]
So it won't hurt our eyes to see it on mobile

**Comment by gl1000007** [comment]
A common use case is entering the site from a smartphone or tablet, for example during parlor meetings and many outdoor activities.
Another use case is adding and editing questions and answers on the go.
The improved design should take those into account.

**Comment by gl1000007** [comment]
@yaninelbi 
I can help with some UX insights. Do you mind if I take a look at the work in progress?

**Comment by yaninelbi** [comment]
Hi guys, I haven't started on this ticket, I will get an initial version
done this Monday after the meetup :-)
You're welcome to join!

**Comment by gl1000007** [comment]
@yaninelbi what's up?
Is there something to see, or we're at square one?
I know it's a dirty word at the workshop, but the two of us will somehow need to coordinate.
I surely can make a wireframe alone, but it's a big task, maybe you have already done something that can be used, and of course 2 minds are better than 1.
I'm available on [github user name] at gmail dot com, and on Github, of course.

**Comment by yaninelbi** [comment]
Hi Gilad! Due to unfortunate events I couldn't attend on Monday :( @daonb
Will make head way next week.

**Comment by yaninelbi** [comment]
If you could be there as well, we can brainstorm and sketch and implement on the spot

**Comment by gl1000007** [comment]
yeah, but until then we can use github, google drive, email, etc...
why wait a week?

**Comment by yaninelbi** [comment]
We'll, I haven't started on it yet so I guess either u upload some wires or sketches or I'll start something and let you know, probably towards the weekend

**Comment by gl1000007** [comment]
Started today making wireframes. It **will** take some time.
First version is transfer of the current design with no changes into [Balsamiq Mockups](http://balsamiq.com/products/mockups/). Think of it as a first commit. When it's ready, I'll share it using Google Drive, where everybody will be able to comment on it.
Then the real fun begins, making it responsive.

Meanwhile, take a look at these pages, listing various ways for Balsamiq to export contents. Maybe something here will save us some time translating wireframes into working pages:
[Importing and Exporting](http://support.balsamiq.com/customer/portal/articles/111730)
[Extensions - Exporting your Mockups to code](http://support.balsamiq.com/customer/portal/articles/135659#exporting)

Update: on hold, currently working on https://github.com/hasadna/open-shot/issues/106

**Comment by daonb** [comment]
@yaninelbi we're launching tonight! are you going to make it?

**Comment by lotan1** [comment]
must change
![screenhunter_64 sep 30 18 33](https://f.cloud.github.com/assets/4595165/1238229/b4307584-29e5-11e3-83fe-2c743941e0e8.jpg)

**Comment by gl1000007** [comment]
1. No time to design responsive spec!
2. There _is_ a layout that makes the change [here (specific bookmark)](https://docs.google.com/document/d/1cEObTn5NWn-BjkWpSiqpsmy8zCMGVmmt4h9rv8XaRnE/edit#bookmark=id.nzrbtpczb2s4). The problem is lack of time to write documentation in Google Drive and break it into issues.
3. Why you are in Netanya and the navbar displays Beer-Sheva? When I tried, I saw maale avak, the former locality I viewed. This demands a separate issue.

**Comment by lotan1** [comment]
we need some how to allow at least the use of Enter

**Comment by tbrisker** [comment]
lotan - you are talking about a different issue - #56 .

**Comment by lotan1** [comment]
תכלס

**Comment by tbrisker** [comment]
Will not change design to responsive until after elections.
